server/helper_test.go

```diff
@@ -449,6 +461,39 @@ func (th *testHelper) assertEphemeralMessage(t *testing.T, userID, channelID, me
 	}
 }
 
+func (th *testHelper) assertNoEphemeralMessage(t *testing.T, userID, channelID, message string, maxWaitTime time.Duration) {
```

Comment:
Neat! I had half a mind to make this the default behaviour at the end of every test, but naturally it's wasteful to wait every time. Thank you!